multi-node/aws/pkg/config/region_map.go

```diff
@@ -0,0 +1,62 @@
+package config
+
+import (
+	"fmt"
+
+	"github.com/coreos/coreos-kubernetes/multi-node/aws/pkg/coreosutil"
+)
+
+var regions = []string{
+	"ap-northeast-1",
+	"ap-southeast-1",
+	"ap-southeast-2",
+	"eu-central-1",
+	"eu-west-1",
+	"sa-east-1",
+	"us-east-1",
+	"us-gov-west-1",
+	"us-west-1",
+	"us-west-2",
+}
+
+var supportedChannels = []string{
+	"alpha",
+	"beta",
```

Comment:
beta will not be supported once this PR is updated to include the kubelet-wrapper